docs/api_reference/messages_api/post_send_messages.md

```diff
@@ -4,6 +4,10 @@ title: POST /messages/send
 
 # POST /messages/send
 
+:::caution
+After 24h without a reply from the customer, it is not possible to send regular messages, you'll need to use a **Template** message, see examples below.
```

Comment:
```suggestion
After 24 hours without a response from the customer, it is not possible to send regular messages. You will need to use a **template message**, as shown in the examples below.
```

Reply:
Also we could link the "example" word to the paragraph where we show how to send templates?

Reply:
ah yes 🆗